back-end/legeno-around-here/src/main/java/wooteco/team/ittabi/legenoaroundhere/domain/post/Post.java

```diff
@@ -92,6 +96,19 @@ public void addPostImages(PostImage postImage) {
         }
     }
 
+    public void addPostImage(PostImage postImage) {
+        if (!this.postImages.contains(postImage)) {
+            this.postImages.add(postImage);
+        }
+        if (!postImage.hasPost()) {
+            postImage.setPost(this);
+        }
+    }
+
+    public void removePostImage(PostImage postImage) {
+        this.postImages.remove(postImage);
+    }
```

Comment:
바깥에서 forEach로 호출하는데, 아예 removeAll로 만드는 건 어떨까요?

```suggestion
    public void removePostImage(List<PostImage> postImages) {
        this.postImages.removeAll(postImages);
    }
```

Reply:
조언 감사합니다! 그게 훨씬 더 깔끔하겠네요! 함수명은 removePostImages로 하겠숩니다!